book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -12,17 +12,17 @@ Samakatuwid, kung ikaw ay naghintay nang sobrang tagal upang i-merge ang dalawan
 Sa seksyon na ito, kami ay magpapatuloy kung ano ang mga maaaring isyu at anong mga kasangkapan na nagbibigay tulong upang panghawakan ang mga ito na mas madayang mga sitwasyon.
 At saka tatalakayin din natin ang iba't iba, na hindi pamantayan na mga uri ng mga merge na maaari mong gawin, pati na rin makita kung paano umatras sa mga merge na iyong nagawa.
 
-==== Merge Conflicts
+==== Pag-merge sa mga Kasalungat
 
-While we covered some basics on resolving merge conflicts in <<_git_branching#_basic_merge_conflicts>>, for more complex conflicts, Git provides a few tools to help you figure out what's going on and how to better deal with the conflict.
+Habang sakop namin ang maraming mga pangunahin na paglulutas ng merge na mga salungat sa <<_git_branching#_basic_merge_conflicts>>, para sa higit na kumplikado na mga salungat, Ang Git ay nagbibigay na ilang mga kasankapan para tulungan ka na tingnan kung ano ang nangyayari at kung paano mas mahusay na makitungo sa mga salungat.
 
-First of all, if at all possible, try to make sure your working directory is clean before doing a merge that may have conflicts.
-If you have work in progress, either commit it to a temporary branch or stash it.
-This makes it so that you can undo *anything* you try here.
-If you have unsaved changes in your working directory when you try a merge, some of these tips may help you lose that work.
+Una sa lahat, kung sa lahat ay posible, subukan na siguraduhin na ang iyong tinatrabahong direktoryo ay malinis bago magsagawa ng merge na maaaring magkakaroon ng mga salungat.
+Kung merong kang trabaho na nasa proseso, alinmang i-commit ito sa isamg pansamantalang branch o itago ito.
+Ginagawa ito nito para maka-undo ka ng *anumang bagay* na sinubukan mo dito.
+Kung meron kang hindi na-save na mga pagbabago sa iyong tintrabahong direktoryo kapag sinubukan mong i-merge, ilan sa mga tip na ito ay tumutulong sayo na mawala ang trabahong iyon.
 
-Let's walk through a very simple example.
-We have a super simple Ruby file that prints 'hello world'.
+Tingnan natin ng mabuti ang pinaka simpleng halimbawa.
```

Comment:
change to `Tingnan nating mabuti ang pinakasimpleng halimbawa.`

Reply:
done change